src/pages/Main/Tealist/Slide/Slide.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Slide.scss';
+
+class Slide extends React.Component {
+  render() {
+    return (
+      <li className="slide">
+        <div className="tea-video">
+          <video className="video" autoplay="autoplay" muted loop>
```

Comment:
autoplay는 bool속성입니다! 여기에 문자열을 할당하는건 맞지 않아보입니다(문자열은 truthy값이니까 동작은 하지만 적합한 타입은 아닙니다)
props 또는 속성을 부여할 때 속성명만 적으면 true 값으로 부여됩니다!
```suggestion
          <video className="video" autoplay muted loop>
```

Reply:
반영했습니다!